test/kernels/test_newton_girard_additive_kernel.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+
+from unittest import TestCase
+from gpytorch.kernels import RBFKernel, ScaleKernel, AdditiveKernel, NewtonGirardAdditiveKernel
+from gpytorch.likelihoods import GaussianLikelihood
+from gpytorch.mlls import ExactMarginalLogLikelihood
+from gpytorch.models import ExactGP
+from gpytorch.means import ConstantMean
+from gpytorch.distributions import MultivariateNormal
+import torch
+
+
+class TestNewtonGirardAdditiveKernel(TestCase):
```

Comment:
There's a `gpytorch.test.BaseKernelTestCase` that you can extend that, if you define the `create_kernel_no_ard` and/or `create_kernel_ard` will also run our standard suite of tests for things like batch mode support and handling active_dims correctly.